src/tests/multihost/alltests/test_inmemory_cache.py

```diff
@@ -0,0 +1,356 @@
+""" Automation of memory cache suite
+
+:requirement: IDM-31530
```

Comment:
There should be probably "inmemory_cache" to link to RHEL-115036 - child requirement of IDM-31530.